string name
string username
uint32 pid
uint32 ppid
string status
float32 cpu_percent
float32 memory_percent
float64 create_time
string exe
string[] cmdline
string terminal
rcp/KeyValue[] environ
string cwd
int32 nice 
int32 ionice
uint32 cpu_num
uint32 num_threads
uint32[] cpu_affinity
uint64 ctx_switches_voluntary
uint64 ctx_switches_involuntary
rcp/ProcessCpuTimes cpu_times
rcp/ProcessMemInfo memory_info
uint32 num_fds
rcp/ProcessIoCounters io_counters
rcp/Thread[] threads
uint32 uid_real
uint32 uid_effective
uint32 uid_saved
uint32 gid_real
uint32 gid_effective
uint32 gid_saved